### PR TITLE
update readme!

### PR DESCRIPTION
**Pull request by ocke**
I updated the readme and deploy files.

No code changes.

### CODE REVIEW

File: DEPLOY.md [hunk 1]
```diff
@@ -1,29 +1,30 @@
-Note: Run commands in the root app directory. And obviously you don't need to run yarn install all the time...
+Note: Run commands in the root app directory. And obviously you don't need to run npm install all the time...
 
 Compile the app for DEV
 ===============
-1) `yarn install`
-3) `yarn watch`
+1) `npm install`
+3) `npm run watch`
```
Comment (Mehul-Porwal) [inline DEPLOY.md]: 2?

File: DEPLOY.md [hunk 2]
```diff
@@ -1,29 +1,30 @@
-Note: Run commands in the root app directory. And obviously you don't need to run yarn install all the time...
+Note: Run commands in the root app directory. And obviously you don't need to run npm install all the time...
```
Comment (Mehul-Porwal) [inline DEPLOY.md]: Add a pre-requisite section for installing `zat`?

File: DEPLOY.md (hunk 1 above)
Comment (ocke) [reply DEPLOY.md]: haha oops

File: DEPLOY.md (hunk 2 above)
Comment (ocke) [reply DEPLOY.md]: ok good point, i will add that

File: DEPLOY.md (hunk 1 above)
Comment (ocke) [reply DEPLOY.md]: I changed the numbers to only `1)` because markdown will autonumber it for us.

File: README.md [hunk 3]
```diff
@@ -3,35 +3,39 @@
...
-This is the [User Data App](https://www.zendesk.com/apps/user-data) for Zendesk. The App displays relevant information about the user and their ticket, fields and organizations
+This is the [User Data App](https://www.zendesk.com/apps/user-data) for Zendesk. The App displays relevant information about the user and their ticket, fields and organizations.
 
 Setup
 ===============
 Since a ZAF V2 app is free to implement its own system, we have tried several approaches and looked at how the [App scaffold](https://github.com/zendesk/app_scaffold) had merged different systems together.
 
 * dist/
-  * An empty folder that needs *no* input from the developer. Point your ZAT server to this folder: `zat server -p dist/`
+  * An empty folder that needs *no* input from the developer. It's contents comes from webpack. Point your ZAT server to this folder: `zat server -p dist/`.
```
Comment (vaevictis) [inline README.md]: it's `its` here :)

**Pull request merged**